employee-rostering-backend/src/test/java/org/optaweb/employeerostering/controller/SkillControllerTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaweb.employeerostering.controller;
+
+//SpringBootTest loads entire application, WebMvcTest only loads controller and its dependencies
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.optaweb.employeerostering.domain.Skill;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+public class SkillControllerTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Test
+    public void getSkillListTest() throws Exception {
+        mvc.perform(MockMvcRequestBuilders
+                            .get("/tenant/{tenantId}/skill", 0)
+                            .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
```

Comment:
With a growing number of tests, increasing console output will become less and less comprehensible. Generally, you want each test to either pass or fail (with a self-explanatory failure message). Such an output should be used only for debugging purposes, so I would recommend using `DEBUG` level logging instead of printing to console output.